### PR TITLE
Make receive method more robust

### PR DESCRIPTION
**Pull request by gregorjerse**

### CODE REVIEW

File: resolwe/test_helpers/test_runner.py [hunk 1]
```diff
@@ -329,6 +329,8 @@ async def _run_on_infrastructure(meth, *args, **kwargs):
                         logger.exception("Exception while running test")
                     finally:
                         zmq_info.authenticator.stop()
+                        # Give authenticator task some time to finish.
```
Comment (dblenkus) [inline resolwe/test_helpers/test_runner.py:332]: Put this into a separate commit.

**Comment by gregorjerse** [comment]
Has been merged but not closed?
